### PR TITLE
IGAPP-828: Fix relative links in documentation

### PR DESCRIPTION
**Pull request by timoludwig**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Review by maxammann — APPROVED** [review]
Nice first PR! :)

File: build-configs/README.md [hunk 1]
```diff
@@ -33,11 +33,11 @@ See the corresponding sections for [native](../native/docs/build-configs.md#usin
 
 ## Structure and Available Options
 
-See the [build config types](./BuildConfigType.js) for more information on structure, available options and their functionality.
+See the [build config types](./BuildConfigType.ts) for more information on structure, available options and their functionality.
 
 ## Technical Information
 
-Each build config is a set of javascript files (possibly including common files) that can be found in the [corresponding directory](../build-configs/configs).
+Each build config is a set of javascript files (possibly including common files) that can be found in the [corresponding directory]().
```
Comment (klinzo) [inline build-configs/README.md:40]: Missing a link here?

**Review by klinzo — APPROVED** [review]
Thanks! Did not test the new links :)

File: build-configs/README.md (hunk 1 above)
Comment (timoludwig) [reply build-configs/README.md:40]: This just links to the relative directory `.` (in this case `build-configs`)

**Comment by timoludwig** [comment]
> Did not test the new links :)

No risk, no fun :innocent:

**Pull request merged**